### PR TITLE
Review 'Media guidelines' page

### PR DESCRIPTION
**Pull request by jdbaldry**
- Include more context in the description
- Remove use of "simple"
- Simplify some language for readability
- Consolidate video sections
- Use consistent placeholder variable notation
- Use a high quality example of alt text

Closes https://github.com/grafana/writers-toolkit/issues/730

### CODE REVIEW

**Review by clayton-cornell — APPROVED** [review]
Looks ok to my eyes

File: docs/sources/write/image-guidelines/index.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Do _not_ use screenshots for the following items:
 
 - Simple create operations, such as create a user, a team, an organization, and so on
 - Primary or secondary navigation items
-- Code samples (show code samples in code blocks)
+- Code samples (instead show code samples in code blocks)
```
Comment (chri2547) [inline docs/sources/write/image-guidelines/index.md:108]:
```suggestion
- Code samples (instead, show code samples in code blocks)
```

**Review by chri2547 — APPROVED** [review]
one nit

File: docs/sources/write/image-guidelines/index.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ Do _not_ use screenshots for the following items:
...
 - Dialog boxes that are easy to understand
 - Message text (instead show message text within the Markdown)
```
Comment (jdbaldry) [inline docs/sources/write/image-guidelines/index.md:110]:
```suggestion
- Message text (instead, show message text within the Markdown)
```

**Pull request merged**